product_extended_segmentation/model/template.py

```diff
@@ -194,24 +197,26 @@ def _factor(factor, product_efficiency, product_rounding):
         # NOTE: Instanciating BOM related product
         product_tmpl_id = tmpl_obj.browse(
             cr, uid, bom.product_tmpl_id.id, context=context)
-        # Use threshold in the next iteration just comparing less than in this
-        # case
-        if price < product_tmpl_id.standard_price:
-            tmpl_obj.message_post(cr, uid, [product_tmpl_id.id],
-                                  'Not Updated Cost, But Segments only.',
-                                  'I cowardly did not update Standard new \n'
-                                  'price less than old price \n'
-                                  'new {new} old {old} \n'
-                                  'Segments where written CHECK AFTERWARDS.'
-                                  '{segments}'.
-                                  format(old=product_tmpl_id.standard_price,
-                                         new=price, segments=str(sgmnt_dict)))
+
+        current_price = product_tmpl_id.standard_price
```

Comment:
LEft the comments around to inform the next programmer that here is the threshold feature.